content/hilfe/Handbuch/mitarbeiter-app/index.md

```diff
@@ -68,13 +75,17 @@ src="installieren-ios-2.jpeg" /> {{< /figure >}}
 ## Einloggen
 
 Nachdem ihr Arbeitergeber Sie wie unter [Zugänge
-&Rechte](/hilfe/handbuch/mitarbeiter/zugaenge-rechte) beschrieben per Email
+& Rechte](/hilfe/handbuch/mitarbeiter/zugaenge-rechte) beschrieben per Email
 eingeladen hat und Sie sich registriert haben, können Sie sich in die
 Mitarbeiterapp einloggen. Hierzu rufen Sie die Mitarbeiterapp unter
 https://perso.pentacode.app auf und geben ihre Email Adresse, sowie das
 selbstgewählte Passwort ein. Worauf Sie in die **Übersicht** der Mitarbeiterapp
 gelangt.
 
+{{<figure caption="" class="float right narrow" >}}
+<img src="übersicht.png" />
+{{</figure>}}
+
```

Comment:
Bei Hochkant-Bildern ist es immer blöd das Bild in der gesamten Breite darzustellen, da das Bild dann vertical viel zu viel Platz einnimmt. Mit dem `class="float right narrow"` wird das Bild smaller dargestellt und der Text kann darum herum fließen. Allerdings muss das Bild, dann früher im Artikel platziert werden (musst einfach ausprobieren, wo es am besten aussieht).